### PR TITLE
Disable survey prompt until the next release

### PR DESCRIPTION
**Pull request by nkubala**
this will disable showing the survey prompt completely until `v1.14.0` is released

### CODE REVIEW

**Comment by tstromberg** [comment]
Shouldn't a test fail?

File: pkg/skaffold/survey/survey.go [hunk 1]
```diff
@@ -62,6 +63,14 @@ func New(configFile string) *Runner {
 }
 
 func (s *Runner) DisplaySurveyPrompt(out io.Writer) error {
+	// TODO(nkubala): remove after v1.14.0 is released
+	currentVersion, err := version.ParseVersion(version.Get().Version)
+	if err == nil {
+		nextRelease, _ := version.ParseVersion("v1.14.0")
```
Comment (tstromberg) [inline pkg/skaffold/survey/survey.go:69]: This should check for error, or use semver.MustParse.

Comment (nkubala) [reply pkg/skaffold/survey/survey.go]: updated

**Comment by nkubala** [comment]
the tests won't fail because i put this logic where the survey actually gets shown, not where we check whether we should show it. this guarantees it will never get show, even if skaffold thinks it should/will.

**Comment by AndiDog** [comment]
Not sure if you found the root cause already, but here's a hint. For v1.13, I fixed this by running `skaffold completion bash >/usr/local/etc/bash_completion.d/skaffold`.

**Pull request merged**